ambari-server/src/main/resources/common-services/HDFS/2.1.0.2.0/metainfo.xml

```diff
@@ -193,7 +193,7 @@
         </osSpecific>
         
         <osSpecific>
-          <osFamily>debian7,ubuntu12,ubuntu14</osFamily>
+          <osFamily>kylin4,debian7,ubuntu12,ubuntu14</osFamily>
```

Comment:
If you plan to also include this in HDP 3.0, then please add kylin4 to the metainfo.xml file for ZK 3.4.9, HDFS 3.0.0.3.0, and YARN 3.0.0.3.0 in common services